### ISSUE TITLE
Low alignment rates (2021 exp)?

### ISSUE THREAD

**Issue by sr320** [grace-ac/paper-pycno-sswd-2021-2022#1]
I am seeing some samples that have quite low alignment rates

![rates](http://gannet.fish.washington.edu/seashell/snaps/Monosnap_MultiQC_Report_2024-05-26_11-44-42.png) 

Do you see the same thing?

**Comment by grace-ac** [comment]
yes,  i'm not sure how to run multiqc on sam files, but i looked at the hisat output files and the following libraries had low alignment rates (ranging 10-24%)   
34, 35, 36, 37, 38, 39, 40,

the remaining libraries were all roughly around 76-80%  alignment, with libraries 75 and 78 having 46% and 63% respectively  

## EDIT: double checked percentages and our numbers are the same!

**Comment by sr320** [comment]
Does not seem to be related to treatment
![](http://gannet.fish.washington.edu/seashell/snaps/Monosnap_RNAseq_library_metadata_2024-05-27_12-19-15.png)

did these happen to be extracted as same batch?

**Comment by grace-ac** [comment]
yes, right about treatment. i'll check my extraction notes for extraction batch date!

**Comment by grace-ac** [comment]
it looks like for my summer 2021 samples i didn't make a note in an obvious place about what batch the samples were extracted in. i'll keep digging through my notes and spreadsheets.
